.azure/pipelines/jobs/default-build.yml

```diff
@@ -137,7 +137,14 @@ jobs:
     - powershell: |
         Write-Host "##vso[task.setvariable variable=SeleniumProcessTrackingFolder]$(BuildDirectory)\obj\selenium\"
         ./eng/scripts/InstallGoogleChrome.ps1
-
+      displayName: Install Chrome
+  - ${{ if eq(parameters.agentOs, 'Linux') }}:
```

Comment:
This should be unnecessary. Chrome is pre-installed in our Linux agents

https://github.com/Microsoft/azure-pipelines-image-generation/blob/master/images/linux/Ubuntu1604-README.md